HIP Documentation/HIP Programming Guide.rst

```diff
@@ -0,0 +1,23 @@
+=========================
+HIP Programing Guide
+=========================
+
+What is this repository for?
+
+HIP allows developers to convert CUDA code to portable C++. The same source code can be compiled to run on NVIDIA or AMD GPUs. 
+
+Key features 
+==============
+
+HIP is very thin and has little or no performance impact over coding directly in CUDA or hcc “HC” mode.
```

Comment:
Is HIP-Clang still supporting "HC" mode? This may be left over from the previous HIP-HCC component tha is now deprecated.